travis.sh

```diff
@@ -49,8 +64,16 @@ function run_docker() {
     echo -e $(colorize BOLD "Starting Docker image: $DOCKER_IMAGE")
     travis_run docker pull $DOCKER_IMAGE
 
+    setup_ssh_keys
+    # Forward ssh agents
+    local -a run_opts
+    local auth_dir
+    auth_dir=$(dirname "$SSH_AUTH_SOCK")
+    run_opts+=(-v "$auth_dir:$auth_dir" -e "SSH_AUTH_SOCK=$SSH_AUTH_SOCK")
```

Comment:
industrial_ci only forwards this in case `$SSH_AUTH_SOCK` is actually set. As ssh-agent is not started by industrial_ci, I'm not sure who actually does.

Reply:
I think Travis is calling `ssh-agent` internally which set these env variables